modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java

```diff
@@ -278,8 +312,32 @@ public void testSessionHijacking() throws Exception {
         HttpResponse response = client.execute(getRequest);
         String resultPage2 = DataExtractUtil.getContentData(response);
         EntityUtils.consume(response.getEntity());
-        Assert.assertTrue(resultPage2.contains("Authentication Error!"), "Session hijacking is possible.");
+        Assert.assertTrue(resultPage2.contains("Authentication Error!"), "Session hijacking is possible for " +
+                "tenant domain: " + tenantDomain);
+    }
+
+    @Test(alwaysRun = true, description = "Test logout request",
+            dependsOnMethods = {"testPassiveSAML2Assertion", "testSessionHijacking"})
+    public void testSendLogoutRequest() throws Exception {
 
+        String passiveParams = "?wa=wsignout1.0&wreply=" + PASSIVE_STS_SAMPLE_APP_URL + "&wtrealm=PassiveSTSSampleApp";
+        passiveParams = appendTenantDomainQueryParam(passiveParams);
+        HttpGet request = new HttpGet(this.passiveStsURL + passiveParams);
+        HttpResponse response = client.execute(request);
+
+        Assert.assertNotNull(response, "PassiveSTSSampleApp logout response is null for tenant domain: " +
+                tenantDomain);
+        int responseCode = response.getStatusLine().getStatusCode();
+        Assert.assertEquals(responseCode, 200, "Invalid Response for tenant domain: " + tenantDomain);
+
+        Map<String, Integer> keyPositionMap = new HashMap<>(1);
+        keyPositionMap.put("name=\"sessionDataKey\"", 1);
+        List<DataExtractUtil.KeyValue> keyValues = DataExtractUtil.extractDataFromResponse(response,
+                keyPositionMap);
+        EntityUtils.consume(response.getEntity());
+        Assert.assertNotNull(keyValues, "sessionDataKey key value is null for tenant domain: " + tenantDomain);
```

Comment:
I feel it would be better to include what we expected to happen here.
"Authentication request was not initiated after logout from sample. Possible logout failure."

Reply:
In addition to the current message.

Reply:
done